MaskRCNN_no_batch_convergence/train.py

```diff
@@ -847,7 +834,7 @@ def do_predict(pred_func, input_file):
             callbacks.append(GPUUtilizationTracker())
 
         if args.perf:
-            callbacks.append(ThroughputTracker(args.images_per_step,
+            callbacks.append(ThroughputTracker(BATCH_SIZE_PLACEHOLDER*cfg.TRAIN.NUM_GPUS,
```

Comment:
steps_per_epoch passed in as an argument to the training script isn't being accounted for here.  Causes the mean epoch throughput to be miscalculated.

Reply:
This is only an issue if we are deliberately passing in an incorrect steps_per_epoch values, right?

Looking at ThroughputTracker, I see 

`def __init__(self, items_per_step, items_per_epoch, trigger_every_n_steps=None, log_fn=None):`

so the code seems correct unless we are trying to have epochs be different than one full cycle of the data.

Reply:
It's more to avoid accidentally not changing the steps_per_epoch at the training script level when changing batch sizes.  Could this be calculated internally?  Not a big deal either way.